client/src/components/Sub_RelatedItems/Sub_SingleProd/ComparisonModal.jsx

```diff
@@ -1,16 +1,67 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 export default function ComparisonModal({
-  thisProduct, parentProduct, theseStyles, thisAvgRating, productStyle, reviewMetadata
+  thisProduct, thisReviewMetadata,
+  parentProduct, parentReviewMetadata,
+  setAllowCardClick, setShowComparisonModal, getAvgRating,
 }) {
+  const thisAvgRating = getAvgRating(thisReviewMetadata)[0];
+  const parentAvgRating = getAvgRating(parentReviewMetadata)[0];
+
+  const characteristicsCompare = [
+    thisReviewMetadata.characteristics, parentReviewMetadata.characteristics
+  ];
+
+  const handleMouseEnter = () => {
+    setAllowCardClick(false);
+  };
+
+  const handleMouseLeave = () => {
+    setAllowCardClick(true);
+    // setShowComparisonModal(false);
+  };
```

Comment:
I think you can just have this in anonymous function inside eventhandlers.

Reply:
Good point. I think yesterday I wondered if I would add more lines of code to each, but, I don't think I need to.